.github/workflows/cron.yml

```diff
@@ -43,6 +43,11 @@ jobs:
     - name: Compact DB
       run: make db-compact
 
+    - name: Compress assets
+      run: make db-compress
+      env:
+        DB_TYPE: trivy-light
```

Comment:
I think you didn't meant this to be here right?

Reply:
oooops. sorry for my mistake

Reply:
No worries! I fixed it: https://github.com/aquasecurity/trivy-db/pull/47/commits/09f3033bd8c8bf621d108feb60de1f87437b803a